scripts/deleteSnapsets.js

```diff
@@ -0,0 +1,162 @@
+/*!
+Copyright 2018 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+// This script modifies the preferences data base:
+// 1. Finds all the Prefs Safes of type "snapset" (prefsSafesType = "snapset"),
+// 2. Finds all the GPII Keys associated with each snapset Prefs Safe
+// 3. Deletes the found Prefs Safes and associated GPII Keys
+//
+// A sample command that runs this script:
+// node deleteSnapsets.js $COUCHDBURL
+
+"use strict";
+
+var http = require("http"),
+    url = require("url"),
+    fluid = require("infusion");
+
+var gpii = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.dataLoader");
+fluid.setLogging(fluid.logLevel.INFO)
+
+var dbLoader = gpii.dataLoader;
+dbLoader.couchDbUrl = process.argv[2];
+if (!fluid.isValue(dbLoader.couchDbUrl)) {
+    fluid.log ("COUCHDB_URL environment variable must be defined");
```

Comment:
Would be useful to also output the command usage: `node deleteSnapsets.js $COUCHDBURL`

Reply:
Done.